bochs/cpu/event.cc

```diff
@@ -138,14 +140,25 @@ void BX_CPU_C::InterruptAcknowledge(void)
     // if no local APIC, always acknowledge the PIC.
     vector = DEV_pic_iac(); // may set INTR with next interrupt
 
-  BX_CPU_THIS_PTR EXT = 1; /* external event */
 #if BX_SUPPORT_VMX
   VMexit_Event(BX_EXTERNAL_INTERRUPT, vector, 0, 0);
 #endif
 
-  BX_INSTR_HWINTERRUPT(BX_CPU_ID, vector,
+#if BX_SUPPORT_UINTR
+  if (BX_CPU_THIS_PTR cr4.get_UINTR() && long64_mode() && vector == BX_CPU_THIS_PTR uintr.uinv)
+  {
+#if BX_SUPPORT_APIC
```

Comment:
Is it actually possible to have a disabled ``BX_SUPPORT_APIC`` flag when ``BX_SUPPORT_UINTR`` is enabled?

Maybe ``config.h.in`` should contain additional sanity checks.

Reply:
Today it is theoretically possible.
Moreover legacy APIC is going to be deprecated by Intel and only X2APIC will survive.
So even if today BX_SUPPORT_APIC=0 makes no sense it might become reasonable soon.